.github/workflows/release.yaml

```diff
@@ -1,11 +1,9 @@
 name: Release
 
 on:
-  # push:
-  #   branches:
-  #     - main
-  # manual trigger
```

Comment:
You should retain the manual trigger just in case (eg if push trigger fails, but you want to retry).